### PR TITLE
🚅  Implementing candidate bullet points 

### PR DESCRIPTION
**Pull request by gyroflaw**

### CODE REVIEW

**Comment by freakstatic** [comment]
LGTM!
@WRadoslaw could you please review?

**Comment by freakstatic** [comment]
Fixes #2793

**Review by WRadoslaw — APPROVED** [review]
LGTM! Thanks 👏

File: packages/ui/src/council/modals/VoteForCouncil/components/CandidacyReview.tsx [hunk 1]
```diff
@@ -33,9 +33,9 @@ export const CandidacyReview = memo(({ candidate, minStake }: CandidacyReviewPro
       <h4>{candidate.info.title}</h4>
 
       <CandidacyPointList>
-        <CandidacyPoint>The More Important the Work, the More Important the Rest</CandidacyPoint>
-        <CandidacyPoint>How to build a loyal community online and offline</CandidacyPoint>
-        <CandidacyPoint>Helping a local business reinvent itself</CandidacyPoint>
+        {candidate.info.bulletPoints.map((bulletPoint) => (
+          <CandidacyPoint>{bulletPoint}</CandidacyPoint>
```
Comment (WRadoslaw) [inline packages/ui/src/council/modals/VoteForCouncil/components/CandidacyReview.tsx:37]:
```suggestion
          <CandidacyPoint key={`${candidate.id}-bullet-${index}`}>{bulletPoint}</CandidacyPoint>
```

**Review by WRadoslaw — CHANGES_REQUESTED** [review]
Realised that there is one thing that's need to be changed :D

**Review by WRadoslaw — APPROVED** [review]

**Comment by dmtrjsg** [comment]
@gyroflaw pls link with originating issue

**Pull request merged**